src/components/CountryDetails.js

```diff
@@ -1,7 +1,32 @@
-// function CountryDetails(){
-//     return (
+import { useParams } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import '../countries.json';
+import axios from 'axios';
 
-//     )
-// }
+function CountryDetails() {
+  const { countryId } = useParams();
+  
+  const [countries, setcountries] = useState([])
 
-// export default CountryDetails;
\ No newline at end of file
+  useEffect(() => {
+      axios.get(`https://ih-countries-api.herokuapp.com/${countryId}`)
+        .then((response) => {
+          setcountries(response.data)
+        })
+        .catch(e => console.log("error getting countries list", e));
+    }, [countryId])
```

Comment:
You are missing word `countries` in your route, instructions gives this as example:
```
 https://ih-countries-api.herokuapp.com/countries/USA
```